modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/client/ItThinClientSqlTest.java

```diff
@@ -213,8 +213,7 @@ void testExecuteAsyncDdlDml() throws InterruptedException {
         assertEquals(3, columns.size());
         assertEquals("MYVALUE", columns.get(0).name());
         assertEquals("ID", columns.get(1).name());
-        // TODO: Uncomment after https://issues.apache.org/jira/browse/IGNITE-19106 Column namings are partially broken
-        //assertEquals("ID + 1", columns.get(2).name());
+        assertEquals("TESTEXECUTEASYNCDDLDML.ID + 1", columns.get(2).name());
```

Comment:
It looks incorrect to have schemaname as part of name of column

Reply:
incorrect in comparison with what ? there is no standard for this case as i know, we can use here all we want and additional table name is a bit informative, why not ?

Reply:
at least it's inconsiste with other aliases in the same query

Reply:
also there is SQL feature T285 related to the behaviour

Reply:
fixed